src/data/list/func.lean

```diff
@@ -30,23 +52,28 @@ variables [inhabited α] [inhabited β]
 
 localized "notation as ` {` m ` ↦ ` a `}` := list.func.set a as m" in list.func
 
+/-- Get element of a list by index. If the index is out of range, return the default element -/
 @[simp] def get : ℕ → list α → α
 | _ []          := default α
 | 0 (a::as)     := a
 | (n+1) (a::as) := get n as
 
+/-- Pointwise equality of lists -/
```

Comment:
Do explain what happens when the two lists aren't the same size. Same for `pointwise`, `add`, `sub`.